Lilu/Sources/kern_devinfo.cpp

```diff
@@ -273,9 +273,17 @@ void DeviceInfo::grabDevicesFromPciRoot(IORegistryEntry *pciRoot) {
 								v.video = pciobj;
 								v.vendor = pcivendor;
 							} else if (pcicode == WIOKit::ClassCode::HDADevice) {
+
```

Comment:
What does amdzen have to do with HDADevice?

Reply:
Nothing.  These are Raven and Zeppelin HD Audio Controllers.